### DISCUSSION TITLE
How do I claim my "swag pack?"

### DISCUSSION THREAD

**Discussion by sdondley** [GothenburgBitFactory/taskwarrior#2753] [Q&A] [answered]
What's the dilly-o?

**Comment by tbabej** [comment, marked as answer]
After 6 month of sponsorship at that tier, sponsor should receive an email with a form :slightly_smiling_face: not too many people at that tier, so the process still might have its flaws. Let me know if that does not happen!

**Comment by sdondley** [comment]
Ok, thanks. Looks I've been at that level for 4 months. I'll be on the lookout for the form. :)
